src/features/embed-gist-inline.js

```diff
@@ -0,0 +1,29 @@
+import select from 'select-dom';
+
+const gistRegex = /gist\.github\.com\/\w*?\/\w*/;
+const isGist = link => gistRegex.test(link.href);
+
+const getJsonUrl = href => `${href}.json`.replace(/\.jso?n?\.json$/, '.json');
```

Comment:
I don't fully understand what `/\.jso?n?\.json$/` is doing. Like, why are we only replacing names that end with `.js` or `.json`? Can you show me an example URL? I think this would be clearer regardless `/\.(js|json)\.json$/`.

Reply:
It is for if the person links specifically the json or js version, so it won't create a link like https://gist.github.com/user/gist.js.json, it's probably unlikely for someone to be linking it but just in case

Reply:
Ok, makes sense. I still would like to use `/\.(js|json)\.json$/` as it's more readable.

Can you also add a short comment above it to explain what it's doing.

Reply:
I don't think `.js` and `.json` links should be embedded, they are technically not the same content (web page vs embed code)